models/products-model.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose')
+const {Schema} = mongoose
+const ProductsSchema =  new Schema({
+  img: 
+      { data: Buffer, contentType: String }
+  , 
+  name : {
+    type : String , 
+    required : true
+  } , 
+  description : {
+    type : String ,
+    required : true
+  }
+    , 
+  price : {
+    type : Number ,  
+    required : true
+  } , 
+   
+  inStock : {
+    type : Boolean ,  
+    required : true
+  } , 
+  fastDelivery : {
+    type : Boolean ,  
+    required : true
+  } ,
+    quantity : {
+    type : Number ,  
+    required : true
+  } 
+}
+)
+
+const Products = mongoose.model('Products' , ProductsSchema , 'products')
```

Comment:
This way you won't have to explicitly pass the third argument like you have to force the name of a collection. Passing `Product` as the first arguement will automatically create "products" collection in you database